portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -34,7 +35,7 @@
     /** Responds with JSON array containing marker data. */
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException{
-        List<Marker> markers = getMarkers();
+        List<Marker> markers = getMarkers(request);
```

Comment:
Not sure why you made this a duplicate of MarkerAreaServlet. We want to reduce duplicate code. At this point, I think it's easier if you just delete MarkerAreaServlet and apply of my comments in that class to here.